README.md

```diff
@@ -1,18 +1,22 @@
+<!-- markdownlint-disable MD010 -->
```

Comment:
https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md010---hard-tabs

What about using spaces instead of tabs and keeping the rule active

Reply:
Got the config working :)